packages/static-module-record/src/babelPlugin.js

```diff
@@ -61,65 +91,133 @@ function makeModulePlugins(options) {
       allowedHiddens.add(id);
     };
 
-    const rewriteVars = (vids, isConst, needsHoisting) =>
-      vids.reduce((prior, id) => {
+    /**
```

Comment:
I posted a follow-up that refactors `markExported` to require less logic by breaking this into `markLiveExport`, `markFixedExport`, and `markExport` (which infers which of the other to use only on whether the named export is initialized more than once). I also comments with my understanding from a close review.

Reply:
That sounds great!  I'll have a look.